db/migrate/20220520205001_create_organizations.rb

```diff
@@ -0,0 +1,14 @@
+class CreateOrganizations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :organizations do |t|
+      t.string :name
```

Comment:
En caso de tener un `NOT NULL` en los requerimientos, deberías tener un `null: false` en el campo correspondiente.

Reply:
Hecho